app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -358,12 +358,13 @@
     <string name="onboarding_button_next">Próximo</string>
     <string name="onboarding_button_ready">Estou pronto!</string>
 
-    <string name="onboarding_welcome_title">Bem-vindo a Monerujo!</string>
-    <string name="onboarding_welcome_information">Esse aplicativo permite que você crie e use carteiras Monero. Você poderá guardar seus queridos moneroj aqui.</string>
+    <string name="onboarding_welcome_title">Bem-vindo à Monerujo!</string>
+    <string name="onboarding_welcome_information">Este aplicativo permite que você crie e use carteiras Monero. Você poderá guardar seus queridos moneros 
+aqui.</string>
```

Comment:
In the English version it's broadly used `moneroj` instead of `moneros` as plural. That comes from Esperanto. I'd rather keep it with `j`.

Reply:
ah, I didn't know that